VictoriaAlbert/VictoriaAlbert/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,142 @@
+                                            <fontDescription key="fontDescription" type="system" pointSize="12"/>
+                                            <nil key="textColor"/>
+                                            <nil key="highlightedColor"/>
+                                        </label>
+                                        <imageView opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" image="default-placeholder" id="yun-vV-P6r">
+                                            <frame key="frameInset" minX="15" width="43" height="43"/>
+                                            <autoresizingMask key="autoresizingMask"/>
+                                        </imageView>
+                                    </subviews>
+                                </tableViewCellContentView>
+                            </tableViewCell>
+                        </prototypes>
+                        <connections>
+                            <outlet property="dataSource" destination="kat-1z-CqM" id="4rn-NW-bdC"/>
+                            <outlet property="delegate" destination="kat-1z-CqM" id="S1L-7z-L0n"/>
+                        </connections>
+                    </tableView>
+                    <navigationItem key="navigationItem" id="3QV-ID-RlT"/>
+                    <connections>
+                        <segue destination="hrT-dk-FgX" kind="show" identifier="detailObjectViewControllerSegue" id="fpX-J8-ehe"/>
+                    </connections>
+                </tableViewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="HFG-Pr-JjN" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="984.79999999999995" y="6.746626686656672"/>
+        </scene>
+        <!--Detail Object View Controller-->
+        <scene sceneID="9TZ-DO-ASb">
+            <objects>
+                <viewController id="hrT-dk-FgX" customClass="DetailObjectViewController" customModule="VictoriaAlbert" customModuleProvider="target" sceneMemberID="viewController">
+                    <layoutGuides>
+                        <viewControllerLayoutGuide type="top" id="Qza-id-bA1"/>
+                        <viewControllerLayoutGuide type="bottom" id="6Mm-Cu-jc0"/>
+                    </layoutGuides>
+                    <view key="view" contentMode="scaleToFill" id="qfR-vg-5BL">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <imageView userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="fcY-dR-ZuP">
```

Comment:
This should be aspectFit or aspectFill in order to prevent the image from distorting.